src/CGRANode.cpp

```diff
@@ -579,3 +579,7 @@ void CGRANode::disableAllFUs() {
   m_supportComplex = false;
   m_supportVectorization = false;
 }
+
+bool CGRANode::isDisabled() {
```

Comment:
So now, `disabled` implicitly indicates two cases:
 - HW choice to disable it, no data can pass it, no operation can map on it, or even serve as a dead point in the fabric.
 - Another simultaneous kernel is occupying this CGRA tile/node, which makes it impossible for current kernel to leverage it.

Correct?

Reply:
Thanks for the confirmation, plz summarize above two cases and put the summary as comment above the `m_disabled`.

Reply:
I think about it again, if CGRANode5 is allocated to `Kernel1`, m_disabled should be set to `false` in `Kernel1`'s `paramCGRA.json` but `true` in other kernels' `paramCGRA.json`